packages/eslint-plugin-pf-codemods/src/rules/helpers/findAncestor.ts

```diff
@@ -0,0 +1,28 @@
+import { JSXElement, JSXOpeningElement } from "estree-jsx";
+
+type ElementWithParent = (JSXOpeningElement | JSXElement) & {
+  parent?: ElementWithParent | null;
+};
+
+export function findElementAncestor(
+  node: ElementWithParent,
+  ancestorNameToFind: string
```

Comment:
Since this helper has only been used to find an element ancestor, tweaked this from the original helper. Kept the file name generic in case we want to add other helpers, like a general "findNodeAncestor" or something.